### DISCUSSION TITLE
What is "threads" in Debugger?

### DISCUSSION THREAD

**Discussion by Chameleon-Coder** [ScratchAddons/ScratchAddons#4412] [Help]
I only use TurboWarp, so, I wonder what Threads means in Debugger.

**Comment by imdevarsh** [comment]
A thread is just a script starting with a hat block. So if I have a when green flag clicked, that is a thread. Or if I have a 'when broadcast received' then it will count as a thread.

**Reply by rimopa** [reply]
I think @Chameleon-Coder means the "Threads" section of the debugger addon.

**Reply by imdevarsh** [reply]
Yes that's what I'm talking about.

**Comment by Chameleon-Coder** [comment]
Okay then.

------- Original Message -------

**Comment by iqnite** [comment]
From the threads tab, you can run scripts step-by-step and show currently running scripts.
